static/_redirects

```diff
@@ -4,7 +4,8 @@
 /blog                                                   /changelog                                          301
 /docs/advanced/features/advanced-external-logins        /docs/advanced/features/external-logins/advanced    301
 /docs/advanced/theme/wysiwyg-legacy                     /docs/advanced/theme/wysiwyg/legacy                 301
-/docs                                                   /docs/welcome	                                    301
+/docs                                                   /docs/welcome	                                      301
+/docs/advanced/theme/analytics                          /docs/category/analytics	                          301
```

Comment:
```suggestion
/docs                                                   /docs/welcome	                                    301
/docs/advanced/theme/analytics                          /docs/category/analytics	                    301
```

Reply:
😛

Reply:
Am I crazy or am I not seeing any diff in your suggestion 🤯 🧑‍🦯🦮